cluster/node-pools/master-default/userdata.yaml

```diff
@@ -455,16 +454,16 @@ write_files:
               -> setPath("/healthz")
               -> disableAccessLog()
               -> "http://127.0.0.1:8080";
-          {{- if or (eq .Cluster.ConfigItems.teapot_admission_controller_service_account_iam "true") (eq .Cluster.ConfigItems.teapot_admission_controller_service_account_iam_userdata "true") }}
             o: Path("/.well-known/openid-configuration")
-              -> setResponseHeader("Content-Type", "application/json")
-              -> static("/.well-known/", "/var/www/openid-configuration/")
-              -> <shunt>;
-            k: Path("/openid-configuration/*")
-              -> setResponseHeader("Content-Type", "application/json")
-              -> static("/openid-configuration/", "/var/www/openid-configuration/")
-              -> <shunt>;
-          {{- end }}
+              -> disableAccessLog()
+              -> "http://127.0.0.1:8080";
+            k: Path("/openid/v1/jwks")
+              -> disableAccessLog()
+              -> "http://127.0.0.1:8080";
+            k2: Path("/openid-configuration/keys.json")
```

Comment:
This forwards to current jwks path (and also advertised path) to the one handled by the apiserver.